src/VirtoCommerce.NotificationsModule.Data/Migrations/20200922134746_AddPredefinedEditedToTemplate.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VirtoCommerce.NotificationsModule.Data.Migrations
+{
+    public partial class AddPredefinedEditedToTemplate : Migration
```

Comment:
This field can be evaluated on the fly in the program logic and I don't see any sense to store it in the db